### PR TITLE
622 set page title for different pages in the app

### LINKED ISSUE #662: Set Page Title for Different Pages within the App
Set page titles - should help with multiple tabs, bookmarks, and so on, we can use the next/head component https://nextjs.org/docs/api-reference/next/head

### PR DESCRIPTION
**Pull request by DesmondSanctity**
Fixes #662

### CODE REVIEW

**Comment by DesmondSanctity** [comment]
@mainawycliffe I have raised a new PR for the page title. Please do check now.

thanks

File: src/pages/LogsPage.tsx [hunk 1]
```diff
@@ -87,65 +88,72 @@ export function LogsPage() {
...
-          paramKey="start"
-          className="w-44 mr-2"
-        />
-      }
-    >
-      <div className="flex flex-col space-y-6 h-full">
-        <div className="flex flex-row items-center w-full">
-          <FilterLogsByComponent />
+    <>
+      <Head prefix={topology?.name ? `Logs - ${topology.name}` : "Logs List"} />
```
Comment (moshloop) [inline src/pages/LogsPage.tsx:92]:
```suggestion
      <Head prefix={topology?.name ? `Logs - ${topology.name}` : "Logs"} />
```

We shouldn't have `List` in any of the titles

File: src/components/SchemaResourcePage/index.tsx [hunk 2]
```diff
@@ -46,41 +47,44 @@ export function SchemaResourcePage({
...
-      </div>
-
-      <Modal
-        open={modalIsOpen}
-        onClose={onClose}
-        bodyClass=""
-        size="full"
-        title={`Create New ${resourceInfo.name}`}
+    <>
+      <Head prefix={resourceInfo ? `Setings - ${resourceInfo.name}` : ""} />
```
Comment (moshloop) [inline src/components/SchemaResourcePage/index.tsx:51]:
```suggestion
      <Head prefix={resourceInfo ? `Settings - ${resourceInfo.name}` : ""} />
```

Comment (DesmondSanctity) [reply src/components/SchemaResourcePage/index.tsx]: This is fixed now

**Comment by mainawycliffe** [comment]
@DesmondSanctity, this looks good; rebase it, and I will merge it.

**Comment by DesmondSanctity** [comment]
@mainawycliffe Done 👍

**Pull request merged**